### PR TITLE
New blog design(issue#2454)

### PR DESCRIPTION
**Pull request by mirzabob**
made button to say "Read blog stories".

### CODE REVIEW

**Comment by mirzabob** [comment]
@amitsin6h i did as was asked by @jywarren. Thanks to inform me.

**Comment by amitsin6h** [comment]
@mirzabob you have changed the link also which is not required. let it be `blog`  only.

File: app/views/home/home.html.erb [hunk 1]
```diff
@@ -101,7 +101,7 @@
     <img class="img img-circle" src="//i.publiclab.org/system/images/photos/000/006/671/original/canoe.jpg"/>
     <h2 class="the-problem"> <%= t('home.home.the_problem.title') %> </h2>
     <h4><%= t('home.home.the_problem.lack_access') %> <a href="/tools"><%= t('home.home.the_problem.tools') %></a> <%= t('home.home.the_problem.need_participation') %></h4>
-    <p><a href="/blog" class="btn btn-primary btn-lg">Read stories &raquo;</a></p>
+    <p><a href="/wiki/stories" class="btn btn-primary btn-lg">Read blog stories &raquo;</a></p>
```
Comment (seafr) [inline app/views/home/home.html.erb]: This route doesn't seem to exist in `config/routes.rb`

**Comment by seafr** [comment]
@mirzabob Is this linked to an issue you have opened?

**Comment by mirzabob** [comment]
No it was kind of asked by @jywarren .

**Comment by seafr** [comment]
I think this is related to issue #2454 as mentioned in the title (sorry, didn't notice that earlier).

**Comment by seafr** [comment]
I think this is related to issue #2454 as mentioned in the title (sorry, didn't notice that earlier).

File: app/views/home/home.html.erb (hunk 1 above)
Comment (seafr) [reply app/views/home/home.html.erb]: @jywarren

**Comment by amitsin6h** [comment]
@mirzabob just change the link, keep it `/blog` else looks good.
We will merge it.

File: app/views/home/home.html.erb (hunk 1 above)
Comment (jywarren) [reply app/views/home/home.html.erb]: True - we may eventually move this into the Features system; but for now this looks great!

**Comment by jywarren** [comment]
This looks super, and my apologies if I got things confused a bit! @mirzabob thanks a lot for your contribution here. Much appreciated! 👍 👍 👍

**Comment by jywarren** [comment]
I made a small tweak and this is complete now! Thanks @mirzabob for sticking with us, and I'm glad you had a chance to go through all the steps of making a contribution! 

We'd love to work with you again if you'd like to take up a new issue!

**Pull request merged**